src/components/color-picker/color-picker.tsx

```diff
@@ -101,151 +105,149 @@ export class ColorPicker implements InteractiveComponent {
     this.internalColorSet(this.color, false, "internal");
   }
 
-  /** When true, hides the hex input */
+  /** When `true`, hides the Hex input. */
   @Prop({ reflect: true }) hideHex = false;
 
-  /** When true, hides the RGB/HSV channel inputs */
+  /** When `true`, hides the RGB/HSV channel inputs. */
   @Prop({ reflect: true }) hideChannels = false;
 
-  /** When true, hides the saved colors section */
+  /** When `true`, hides the saved colors section. */
   @Prop({ reflect: true }) hideSaved = false;
 
   /**
-   * Label used for the blue channel
+   * Accessible name for the RGB's blue channel.
```

Comment:
> Accessible name for the RGB section's blue channel.

Same comment for the ones below... 
`RGB's` --> `RBG section's`
`HSV's` --> `HSV section's`